docs/getting_started.rst

```diff
@@ -59,227 +56,207 @@ view an inventory of objects contained in the Document, simply
-
-Based on our inspection of objects contained in the Document above, we can see that these objects were all created in the namespace ``http://sbols.org/CRISPR_Example``. Thus, in order to take advantage of SBOL-compliant URIs, we set an environment variable that configures this namespace as the default. In addition we set some other configuration options.
+Based on our inspection of objects contained in the Document above, we can see that these objects were all created in the namespace ``http://sbolstandard.org/testfiles``. Thus, in order to take advantage of SBOL-compliant URIs, we set an environment variable that configures this namespace as the default.
 
 .. code:: python
 
-    >>> setHomespace('http://sbols.org/CRISPR_Example')
+    >>> sbol3.set_namespace('http://sbolstandard.org/testfiles')
 
 .. end
 
-Setting the Homespace has several advantages. It simplifies object creation and retrieval from Documents. In addition, it serves as a way for a user to claim ownership of new objects. Generally users will want to specify a Homespace that corresponds to their organization's web domain.
+Setting the namespace has several advantages. It simplifies object creation and retrieval from Documents. In addition, it serves as a way for a user to claim ownership of new objects. Generally users will want to specify a namespace that corresponds to their organization's web domain.
 
 --------------------------
 Creating SBOL Data Objects
 --------------------------
 
-Biological designs can be described with SBOL data objects, including both structural and functional features.  The principle classes for describing the structure and primary sequence of a design are ComponentDefinitions, Components, Sequences, and SequenceAnnotations. The principle classes for describing the function of a design are ModuleDefinitions, Modules, Interactions, and Participations. Other classes such as Design, Build, Test, Analysis, Activity, and Plan are used for managing workflows.
+Biological designs can be described with SBOL data objects, including both structural and functional features.  
+The principle classes for describing the structure and primary sequence of a design are ``Component``, ``Sequence``, ``SequencFeature``, and ``SubComponent``. 
+The principle classes for describing the function of a design are ``Component``, ``SubComponent``, ``Interaction``, and ``Participation``. 
 
-In the official SBOL specification document, classes and their properties are represented as box diagrams. Each box represents an SBOL class and its attributes. Following is an example of the diagram for the ComponentDefinition class which will be referred to in later sections. These class diagrams follow conventions of the Unified Modeling Language.
+.. TODO: delete or edit; Is there a pySBOL3 equivalent to "Design, Build, Test, Analysis, Activity, and Plan"
+    Other classes such as Design, Build, Test, Analysis, Activity, and Plan are used for managing workflows.
```

Comment:
I'd suggest just leaving these out for now.

Reply:
👍

Reply:
```suggestion
```